src/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsLinkedAccounts.tsx

```diff
@@ -148,10 +148,12 @@ const SettingsEditSettingsLinkedAccountsButton: FC<SettingsEditSettingsLinkedAcc
     } catch (err) {
       console.error(err)
 
+      const error = Array.isArray(err) ? err[0] : err
```

Comment:
Snuck this in; last piece of [GRO-1541](https://artsyproduct.atlassian.net/browse/GRO-1541)